agr_api/src/main/java/org/alliancegenome/api/service/DiseaseESService.java

```diff
@@ -76,6 +79,35 @@ public JsonResultResponse<GeneDiseaseAnnotationDocument> getRibbonDiseaseAnnotat
 		return ret;
 	}
 
+	public JsonResultResponse<GeneDiseaseAnnotationDocument> getRibbonDiseaseAnnotationsWithoutPrimaryAnnotations(String focusTaxonId, List<String> geneIDs, String termID, Pagination pagination, boolean excludeNegated, boolean debug) {
+
+		// unfiltered query
+		BoolQueryBuilder query = getBaseQuery(geneIDs, termID, excludeNegated, "gene_disease_annotation", true);
+
+		JsonResultResponse<GeneDiseaseAnnotationDocument> ret = new JsonResultResponse<>();
+		ret.setSupplementalData(getSupplementalData(focusTaxonId, true, debug, query));
+
+		// add table filter
+		addTableFilter(pagination, query);
+		SearchResponse searchResponse = getSearchResponse(query, pagination, getAnnotationSorts(focusTaxonId, debug), false);
+		ret.setTotal((int) searchResponse.getHits().getTotalHits().value);
+
+		List<GeneDiseaseAnnotationDocument> list = Arrays.stream(searchResponse.getHits().getHits())
+			.map(searchHit -> {
+				try {
+					GeneDiseaseAnnotationDocument object = mapper.readValue(searchHit.getSourceAsString(), GeneDiseaseAnnotationDocument.class);
+					object.setUniqueId(searchHit.getId());
+					object.setPrimaryAnnotations(null);
```

Comment:
Can you re-use the method that returns the objects with primary annotations and then st the primaryAnnotations attribute to null? that way you don't have to maintain two almost equal emthods.

Reply:
Ah, sure that's a good idea.